test/gtest/uct/ib/test_ib_pkey.cc

```diff
@@ -214,6 +214,48 @@ UCS_TEST_P(test_uct_ib_pkey, test_pkey_pairs) {
                                                  (uct_device_addr_t*)ib_addr1,
                                                  NULL));
 
+        const uct_iface_is_reachable_params_t params1      = {
+            .field_mask         = 0,
```

Comment:
`field_mask` must be always initialized with corresponding bits, `0` means that no one field is initialized.

Reply:
Sorry, I miss that in test code. Done. Thanks

Reply:
Please note that it must reflect in implementation, this test should not pass with correct implementation

Reply:
Done. Remove NULL pointer testcase.